### PR TITLE
Unbreak CI wheel building on Linux

### PR DESCRIPTION
**Pull request by madig**
Edit: So, the 2.0 release was botched. The bundling didn't work on Linux because the FREETYPEPY_BUNDLE_FT variable didn't reach the Docker container. This PR makes it work. AppVeyor deployment may need a separate fix.

### CODE REVIEW

**Comment by madig** [comment]
Something's wrong. The Linux bundling jobs do not bundle the library and the non-bundling job does not create a universal wheel. The macOS build seems to work as expected.

**Comment by madig** [comment]
I forgot that the env var might not reach the test. Hm.

**Comment by madig** [comment]
Slowly getting there... why did this not turn up before?!

**Comment by madig** [comment]
Not really happy with creating a custom Docker wrapper for Linux, but creating an env_vars.sh file on the fly doesn't work... may change it again when I get feedback on https://github.com/matthew-brett/multibuild/issues/183.

File: ci/custom_docker.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    #     UNICODE_WIDTH (optional)
+    #     BUILD_DEPENDS (optional)
+    #     DOCKER_IMAGE (optional)  
+    #     MANYLINUX_URL (optional)
+    #     WHEEL_SDIR (optional)
+    local plat=$1
+    [ -z "$plat" ] && echo "plat not defined" && exit 1
+    local build_cmds="$2"
+    local docker_image=${DOCKER_IMAGE:-quay.io/pypa/manylinux1_\$plat}
+    docker_image=$(eval echo "$docker_image")
```
Comment (belluzj) [inline ci/custom_docker.sh]: What does this line do?

Comment (madig) [reply ci/custom_docker.sh]: Dunno 😁

**Comment by anthrotype** [comment]
I haven't deeply looked into this, but honestly, I don't like that you have to basically "fork" parts from multibuild to accomplish such a basic task of passing an environment variable to the docker container. We should ask multibuild maintainers how to best do this.

**Comment by madig** [comment]
I did: https://github.com/matthew-brett/multibuild/issues/183

Will change once I hear from them.

**Comment by anthrotype** [comment]
https://github.com/rougier/freetype-py/releases/tag/2.0.0.post1

for the title of the Github release notes, it think it's better to keep the same format as the previous ones: e.g. "Version 2.0.0.post1"

**Pull request merged**